### DISCUSSION TITLE
Aspect Label and Accuracy BERTopic Model

### DISCUSSION THREAD

**Discussion by rizkiamandaputri** [MaartenGr/BERTopic#1509] [Q&A] [answered]
Hello again, Marteen!
How to extract aspect label of the topics and how to calculate the accuracy of the model?
Thank's before...

**Comment by MaartenGr** [comment, marked as answer]
You can extract the aspect label of the topics with `topic_model.get_topic(1, full=True)` or with `topic_model.get_topics(full=True)`. The accuracy of the model depends on the evaluation metric. It can be topic coherence, topic diversity, cluster metrics like RAND index, accuracy if its supervised, etc.
